bHY6yh-d1e792/cod-acbnn_bHY6yh-d1e792.xml

```diff
@@ -0,0 +1,197 @@
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e291">
+               <ref xml:id="bHY6yh-d1e792-Rd1e314">Ibidem c. iii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e293">Que 
+               <lb ed="#A"/>desursum est sapientia primum quidem pudica 
+               <lb ed="#A"/>est / deinde pacifica modesta suadibilis 
+               <lb ed="#A"/>bonis consentiens: plena misericordia et fructibus bonis 
+               <lb ed="#A"/>iudicans sine simulatione.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e305">    
+                <lb ed="#A"/><ref xml:id="bHY6yh-d1e792-Rd1e334">Paulus ad corinth. c. xv.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e309">Gratia dei sum id 
+                <lb ed="#A"/>quod sum.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e315">
+               // <ref xml:id="bHY6yh-d1e792-Rd1e345">Idem ad ephesios. ca. ii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e317">Gratia estis 
+               <lb ed="#A"/>saluati per sidem. et hoc non ex vobis dei donum 
+               <lb ed="#A"/>est non ex operibus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e327">
+               // <ref xml:id="bHY6yh-d1e792-Rd1e360">Idem sia ad corinth. 
+               <lb ed="#A"/>c. iii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e329">Sufficientia nostra ex deo est.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e337">
+               <ref xml:id="bHY6yh-d1e792-Rd1e373">Idem
+               <lb ed="#A"/>ad hebreos. iiii. c.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e339">Adeamus cum fiducia ad
+               <lb ed="#A"/> thronum gratie: vt misericordiam consequamur et gratiam inueni<lb ed="#A" break="no"/>amus 
+               in auxilio<!--is that really the word here?-->--> opportuno.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e349">     
+               <lb ed="#A"/><ref xml:id="bHY6yh-d1e792-Rd1e391">Augustinus in Epigramatibus prosperi c. xxvii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e353">
+               <lb ed="#A"/>Nemo eis tam eruditus / nemo eis tam
+               <lb ed="#A"/>doctus qui superna illustratione non egeat.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e362">          
+                <lb ed="#A"/><ref xml:id="bHY6yh-d1e792-Rd1e407">Cassiodorus epistolarum. l. viiiio. c. xxiiii</ref> <quote xml:id="bHY6yh-d1e792-Qd1e366">Deo
+                <lb ed="#A"/>auctore omnia prosperantur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e372">
+               <ref xml:id="bHY6yh-d1e792-Rd1e417">Idem xxv et. fi.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e374">
+               <lb ed="#A"/>Ad omnia redditur habilis quem imbuit doc<lb ed="#A" break="no"/>trina 
+              celestis.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e382">
+               <quote xml:id="bHY6yh-d1e792-Qd1e384"> Idem l.x. c. primo. Dominus co<lb ed="#A" break="no"/>nseuit
```

Comment:
this be moved into a new `<ref>` before the quote, like so `<ref>Idem l.x. c. primo.</ref> <quote>Dominus...`